src/HypothesisTests.jl

```diff
@@ -101,6 +108,9 @@ end
 # parameter of interest: name, value under h0, point estimate
 population_param_of_interest{T<:HypothesisTest}(test::T) = ("not implemented yet", NaN, NaN)
 
+# is the test one- or two-sided
+tail_of_distribution{T<:HypothesisTest}(test::T) = :undefined
```

Comment:
That's a good point. I just followed the definition of `population_param_of_interest`. Is it because there is only one function argument?

I will change the definition together with the function name.